core/chains/evm/client/helpers_test.go

```diff
@@ -38,7 +38,7 @@ func NewClientWithTestNode(t *testing.T, cfg NodeConfig, rpcUrl string, rpcHTTPU
 	}
 
 	lggr := logger.TestLogger(t)
-	n := NewNode(cfg, lggr, *parsed, rpcHTTPURL, "eth-primary-0", id, chainID)
+	n := NewNode(cfg, lggr, *parsed, rpcHTTPURL, "eth-primary-0", id, chainID, 0)
```

Comment:
i think the changelog says the range is 1-100. why is 0 used here?

Reply:
Fixed in [`2c3b889` (#9329)](https://github.com/smartcontractkit/chainlink/pull/9329/commits/2c3b889363a9a0274b0163340157234ab4b86162)